backend/controllers/session.js

```diff
@@ -165,3 +165,7 @@ module.exports.loginStudent = async (req, res) => {
         student: student
     });
 };
+
+module.exports.getInfo = async (req, res) => {
+    return res.status(200).json(await Teacher.findById(req.user._id));
```

Comment:
Can we make sure this is in the same format as all the other JSON responses? 
```suggestion
    return res.status(200).json({
        status: 'ok', 
        teacher: await Teacher.findById(req.user._id)
    });
```